requirements.txt

```diff
@@ -1,3 +1,4 @@
-fondant==0.9.0
+fondant[component,aws,azure,gcp,docker]==0.10.1
```

Comment:
Hmm, we might need this in the other use cases as well. Didn't run into them, but might be because I already had the dependencies in my environment. Will try to re-run with a clean one.